Bash_scripts/HW21/in_hw20/roles/hw201/tasks/centos.yml

```diff
@@ -0,0 +1,11 @@
+- name: "Install programs on CentOS"
```

Comment:
Forget to delete?

Reply:
Sorry, I forgot to delete, everything is fixed.